extensions/import/src/wizard/pages/deployConfigPage.ts

```diff
@@ -172,7 +172,8 @@ export class DeployConfigPage extends DacFxConfigPage {
 		}
 		let values = await this.getDatabaseValues();
 
-		if (this.model.database === undefined) {
+		// set the database to the first dropdown value if upgrading, otherwise it should get set to the textbox value
```

Comment:
nit: in ADS repo, comments go directly after //

Reply:
so it should be like //set... instead of // set... ?

Reply:
Yes, that's what Karl commented for one of my early PRs, But I did notice there are a lot of places we have a space in between.

Reply:
ok I changed the comments in this file to remove the space